torch/csrc/jit/passes/quantization.cpp

```diff
@@ -611,23 +611,21 @@ class InsertQuantDeQuantHelper {
   void run(
     script::Module& module,
     const std::string& method_name);
-  // quantization parameters and scalar type
-  std::tuple<IValue, IValue> getQParams(script::Module& module, Value* v);
+  // quantization parameters including scale, zero_point,
+  // scalar_type and axis(for per channel quantization)
+  std::unordered_map<std::string, IValue> getQParams(script::Module& module, Value* v);
```

Comment:
To make things cleaner it might be good to add a `using` declaration for this `std::unordered_map<std::string, IValue>` type